repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/samlpolicy/SamlFlow10Test.groovy

```diff
@@ -83,10 +78,10 @@ class SamlFlow10Test extends ReposeValveTest {
     }
 
     @Unroll
-    def "a saml:response with an Issuer that #isIt in the configured policy-bypass-issuers list will get an 'Identity-API-Version' value of #headerValue in the request to the origin service"() {
+    def "a saml:response with a(n) '#contentType' type body and an Issuer that #isIt in the configured policy-bypass-issuers list will get an 'Identity-API-Version' value of #headerValue in the request to the origin service"() {
```

Comment:
No need to change anything, but the parenthesis in "a(n)" seems unnecessary since both cases start with `application/`.